packages/appstore/src/types/common.types.ts

```diff
@@ -63,6 +63,9 @@ export type TDetailsOfEachMT5Loginid = DetailsOfEachMT5Loginid & {
         };
     platform?: TPlatform;
     openFailedVerificationModal?: (from_account: string) => void;
+    is_open_order_position_status_present?: boolean;
+    open_order_position_status: boolean;
```

Comment:
can u check the type of this open_order_position_status. It can be undefiened also.. isnt?